src/components/Select/Select.js

```diff
@@ -54,6 +56,11 @@ Select.propTypes = {
   help: PropTypes.string,
   id: PropTypes.string,
   label: PropTypes.string,
+  /**
+   * Optional function that is called when the selection is changed.
+   * The function gets the selected option as an argument.
+   */
+  onSelectionChange: PropTypes.func,
```

Comment:
This comment is a bit redundant, propTypes that are `required` are explicitly marked as such.